### PR TITLE
core: export editor as AMD & CommonJS module via npm, bower and cdn

### PR DESCRIPTION
**Pull request by arekkas**
@inyono npm run tasks `build:lib` and `build:umd` and `build:umd:min` have been added. Please check if this makes sense!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,11 @@
     "docs": "documentation build ./src/editor/**/*.js --format md --github > API.md",
     "docs:watch": "documentation serve ./src/editor/**/*.js --watch",
     "heroku-postbuild": "npm run build:home",
-    "clean": "rimraf \"public/*\"",
-    "build": "cross-env NODE_PATH=. NODE_ENV=production webpack",
+    "clean": "rimraf \"public/*\" \"dist/*\" \"lib/*\"",
+    "build": "npm-run-all clean build:umd:min build:umd build:lib",
```
Comment (inyono) [inline package.json]: Would change that to

```.json
"prebuild": "npm run clean",
"build": "npm-run-all --parallel build:*"
```

File: package.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
   "name": "ory-editor",
   "version": "0.0.0",
   "description": "The next generation platform for creating well-defined, responsive, digital content - as easy as social media.",
-  "main": "public/bundle.js",
+  "main": "lib/editor/index..js",
   "directories": {
-    "bin": "./public",
+    "bin": "./dist",
```
Comment (inyono) [inline package.json]: What is `bin` doing? I thought this is for CLI tools?

File: package.json (hunk 1 above)
Comment (arekkas) [inline package.json]: That makes sense, thanks!

File: package.json (hunk 2 above)
Comment (arekkas) [inline package.json]: Ah, I didn't know that

**Pull request merged**